lib/registry.js

```diff
@@ -424,18 +428,25 @@ registry.autoAttachModel = function(ModelCtor) {
   }
 };
 
-registry.DataSource = DataSource;
-
-/*
- * Core models
- * @private
+/**
+ * Resolve a string or `Model` constructor into a `Model` constructor.
+ * @param {String|Function} name
+ * @returns {Function} The `Model` constructor
  */
 
-registry.Model = require('./model');
-registry.PersistedModel = require('./persisted-model');
+Registry.prototype.resolveModel = function(name) {
+  switch (typeof name) {
+    case 'string':
+      return registry.findModel(name);
```

Comment:
I think we should have two methods: `tryResolveModel` that returns null/undefined when the string name is not found, and `resolveModel` that throws when the string name is not found. Thoughts?